src/battle/partner/lakilester.c

```diff
@@ -22,6 +68,13 @@ INCLUDE_ASM(s32, "battle/partner/lakilester", func_80239140_70CE50);
 
 INCLUDE_ASM(s32, "battle/partner/lakilester", func_80239F84_70DC94);
 
-INCLUDE_ASM(s32, "battle/partner/lakilester", func_8023A19C_70DEAC);
+ApiStatus func_8023A19C_70DEAC(ScriptInstance* script, s32 isInitialCall) {
+    return (D_8023D330 == 0) * 2;
+}
 
 INCLUDE_ASM(s32, "battle/partner/lakilester", func_8023A1B0_70DEC0);
+/* Should work, but there seems to be an issue with the split?
```

Comment:
Ideally this would be a nonmatching instead of commented out code - but we should just fix the split